src/icons/logo.tsx

```diff
@@ -33,7 +33,7 @@ export const LogoIcon = ({ id, className }: LogoIconProps) => {
       <path d="M169.089 0L91 111.991H115.481L193.495 0H169.089Z" fill={fill} />
       <path
         d="M115.5 0L140 34L126.5 53L90 0H115.5Z"
-        fill={`url(#${id ? `${id}_logo_gradient_2` : 'paint2_linear'})`}
+        fill={`url(#${id ? `${id}_logo_gradient_0` : 'paint0_linear'})`}
```

Comment:
[q] what does gradient/paint 0 do vs 2?

Reply:
ah they're diff definitions of colors! i.e. you can expand it out in ur dev console

<img width="1625" alt="Screenshot 2024-07-03 at 4 18 50 PM" src="https://github.com/dydxprotocol/v4-web/assets/70078372/af634457-a4ff-4b09-a0c7-16e1e264caa5">

Reply:
tbh we should be defining this in the svg itself (I think it's falling back to a def from another svg file rn, lemme look into this)